server/webapp/WEB-INF/rails.new/app/presenters/api_v1/admin/merged_environments/merged_environment_config_representer.rb

```diff
@@ -0,0 +1,84 @@
+##########################################################################
+# Copyright 2017 ThoughtWorks, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+##########################################################################
+
+module ApiV1
+  module Admin
+    module MergedEnvironments
+      class MergedEnvironmentConfigRepresenter < ApiV1::BaseRepresenter
+        alias_method :environment, :represented
+
+        link :self do |opts|
+          opts[:url_builder].apiv1_admin_merged_environment_show_url(name: environment.name)
+        end
+
+        link :doc do |opts|
+          'https://api.gocd.io/#merged-environment-config'
```

Comment:
we treat it as a separate endpoint, hence i would like to keep the docs as well seperately.
Though, `environment-config` docs will have references to `merged-environment-config`.